### ISSUE TITLE
Bug: Context Menu right click scrolls grid to top.

### ISSUE THREAD

**Issue by allen-k** [inovua/reactdatagrid#137]
Latest version of community edition and within the documentation and example pages:

Have the grid display enough rows so that you need to scroll down.
Right click on a row to display the context menu and the grid will scroll to the top.
note* very very rarely the grid doesn't scroll, but you're almost certainly guaranteed for it to do so.

**Comment by yurigenin** [comment]
```
import React from 'react'
import ReactDataGrid from '@inovua/reactdatagrid-enterprise'
import '@inovua/reactdatagrid-enterprise/index.css'

import people from './people'
import flags from './flags'

const gridStyle = { minHeight: 250 };

const columns = [
  { name: 'id', header: 'Id', defaultVisible: false, defaultWidth: 80, type: 'number' },
  { name: 'name', header: 'Name', defaultFlex: 1 },
  { name: 'country', header: 'Country', defaultFlex: 1,
    render: ({ value })=> flags[value]? flags[value] : value
  },
  { name: 'city', header: 'City', defaultFlex: 1 },
  { name: 'age', header: 'Age', defaultFlex: 1, type: 'number' }
];

const App = () => {
  const renderRowContextMenu = (menuProps, { rowProps, cellProps }) => {
    menuProps.autoDismiss = true
    menuProps.items = [
      {
        label: 'Row ' + rowProps.rowIndex
      },
      {
        label: 'Want to visit ' + (cellProps.id === 'city'? rowProps.data.city : rowProps.data.country) + '?'
      }
    ]
  }

  return (
    <div>
      <h3>Grid with row context menu</h3>
      <ReactDataGrid
        idProperty="id"
        style={gridStyle}
        renderRowContextMenu={renderRowContextMenu}
        columns={columns}
        dataSource={people}
      />
    </div>
  );
}

export default () => <App />
```

**Comment by allen-k** [comment]
I found a temporary workaround by setting:
 
enableKeyboardNavigation=false

https://reactdatagrid.io/docs/row-selection#row-selection

**Comment by inovua-admin** [comment]
Fixed in `v4.1.17`
